testData/preferences/mary.json

```diff
@@ -4,10 +4,10 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
+                    "http://registry.gpii.net/common/onscreenKeyboard": true,
```

Comment:
I know, and I do agree with you, I was just following the _naming_ conventions for common terms [here](https://github.com/GPII/universal/blob/master/testData/ontologies/ISO24751-flat.json)

@kaspermarkus, are you ok with performing a sed -i s/onscreenKeyboard/onScreenKeyBoardEnabled ?